drivers/hmis_csv_twenty_twenty_six/app/models/hmis_csv_twenty_twenty_six/exporter/export_concern.rb

```diff
@@ -140,6 +140,7 @@ def process(row)
       row = assign_export_id(row)
       row = self.class.adjust_keys(row, @options[:export])
       row = enforce_lengths(row)
+      row = sanitize_string_fields(row)
```

Comment:
should we check lengths after sanitizing?
```suggestion
      row = sanitize_string_fields(row)
      row = enforce_lengths(row)
```